### PR TITLE
Refactor dropdown

### PR DESCRIPTION
**Pull request by AUlasen**
Dropdown  refactored according the last discussion

### CODE REVIEW

File: jdi-light-bootstrap/src/main/java/com/epam/jdi/light/ui/bootstrap/elements/composite/BootstrapDropdown.java [hunk 1]
```diff
@@ -37,7 +34,8 @@ public void collapse() {
 
     @JDIAction("Check that '{name}' is expanded")
     public boolean isExpanded() {
-        return hasClass("show");
+        //return hasClass("show");
```
Comment (RomE-HK) [inline jdi-light-bootstrap/src/main/java/com/epam/jdi/light/ui/bootstrap/elements/composite/BootstrapDropdown.java]: Is these comment still needed?

File: jdi-light-bootstrap/src/main/java/com/epam/jdi/light/ui/bootstrap/elements/composite/BootstrapDropdown.java [hunk 2]
```diff
@@ -46,29 +44,37 @@ public boolean isCollapsed() {
...
+    public BootstrapDropdownAssert shouldBe() {
+        return is();
     }
 
+    @Override
+    public BootstrapDropdownAssert verify() {
+        assertSoft();
+        return is();
+    }
+
```
Comment (RomE-HK) [inline jdi-light-bootstrap/src/main/java/com/epam/jdi/light/ui/bootstrap/elements/composite/BootstrapDropdown.java]: Empty line

File: jdi-light-bootstrap/src/main/java/com/epam/jdi/light/ui/bootstrap/elements/composite/BootstrapDropdown.java [hunk 3]
```diff
@@ -1,23 +1,20 @@
...
-import com.epam.jdi.light.ui.bootstrap.asserts.DropdownAssert;
+import com.epam.jdi.light.ui.bootstrap.asserts.BootstrapDropdownAssert;
 import com.epam.jdi.light.ui.bootstrap.elements.common.Button;
 
-import java.util.Arrays;
-import java.util.List;
+import static com.epam.jdi.light.asserts.core.SoftAssert.assertSoft;
 
-public class Dropdown extends UIBaseElement<DropdownAssert> {
+public class BootstrapDropdown extends Section {
```
Comment (RomE-HK) [inline jdi-light-bootstrap/src/main/java/com/epam/jdi/light/ui/bootstrap/elements/composite/BootstrapDropdown.java]: Our dropdown didn't implement any interfaces and extends regular Section. Don't we need at all methods like select(), list(), etc?

File: jdi-light-bootstrap/src/main/java/com/epam/jdi/light/ui/bootstrap/elements/composite/BootstrapDropdown.java (hunk 1 above)
Comment (AUlasen) [reply jdi-light-bootstrap/src/main/java/com/epam/jdi/light/ui/bootstrap/elements/composite/BootstrapDropdown.java]: Deleted

File: jdi-light-bootstrap/src/main/java/com/epam/jdi/light/ui/bootstrap/elements/composite/BootstrapDropdown.java (hunk 2 above)
Comment (AUlasen) [reply jdi-light-bootstrap/src/main/java/com/epam/jdi/light/ui/bootstrap/elements/composite/BootstrapDropdown.java]: Done

File: jdi-light-bootstrap/src/main/java/com/epam/jdi/light/ui/bootstrap/elements/composite/BootstrapDropdown.java (hunk 3 above)
Comment (AUlasen) [reply jdi-light-bootstrap/src/main/java/com/epam/jdi/light/ui/bootstrap/elements/composite/BootstrapDropdown.java]: BootstrapDropdown is just a section. We have another one class  `DropdownMenu extends BootstrapDropdown` with such methods.

**Comment by AUlasen** [comment]
Issue: https://github.com/jdi-testing/jdi-light/issues/1117

File: jdi-light-bootstrap/src/main/java/com/epam/jdi/light/ui/bootstrap/elements/composite/BootstrapDropdown.java (hunk 3 above)
Comment (RomE-HK) [reply jdi-light-bootstrap/src/main/java/com/epam/jdi/light/ui/bootstrap/elements/composite/BootstrapDropdown.java]: Oh, okay

**Pull request merged**